app/scripts/contentscript.js

```diff
@@ -205,6 +283,7 @@ const setupPageStreams = () => {
 
 const setupExtensionStreams = () => {
   extensionPort = browser.runtime.connect({ name: CONTENT_SCRIPT });
+  // extensionPort.onMessage.addListener(extensionStreamMessageListener);
```

Comment:
Dead code
```suggestion
```

Reply:
see https://github.com/MetaMask/metamask-extension/pull/16075#discussion_r1011961838 and commit to remove comment [1b9e298](https://github.com/MetaMask/metamask-extension/pull/16075/commits/1b9e2983756c77d1fffb16ba20ae77b757eb7488)

(sorry! missed out on adding your suggestion as a co-author here)